app/components/Text.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { StyleProp, StyleSheet, Text as DefaultText, TextStyle } from 'react-native'
+
+export enum FontFaces {
+  LightFont = 'LightFont',
+  RegularFont = 'RegularFont',
+  MediumFont = 'MediumFont',
+  SemiBoldFont = 'SemiBoldFont',
+  BoldFont = 'BoldFont'
+}
+
+export type FontWeight = 'light' | 'regular' | 'medium' | 'semibold' | 'bold'
+
+export const HeaderFont = { headerTitleStyle: { fontFamily: FontFaces.SemiBoldFont } }
```

Comment:
can this be automatically be handled by tailwind within the Navigator that needs it?